docs/guest.md

```diff
@@ -0,0 +1,40 @@
+# Tanzu Kubernetes Grid Service
+This document discusses the velero vSphere plugin installation process in a Tanzu Kubernetes Grid Service environment
+aka as Guest Cluster. For more information on Tanzu Kubernetes Grid Service refer to their [documentation](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html).
+
+## Compatibility
+| vSphere/ESXi Version       | vSphere CSI Version                                                | Kubernetes Version | Velero Version    | Velero Plugin for vSphere Version |
+|----------------------------|--------------------------------------------------------------------|--------------------|-------------------|-----------------------------------|
+| vSphere 7.0P02/ESXi 7.0P02 | Driver version is not v2.1.0, but feature is equivalent to v2.1.0. | v1.16-v1.19        | v1.5.1 and higher | v1.1.0 and higher                 |
```

Comment:
Driver version is not v2.1.0, but feature is equivalent to v2.1.0. -> CSI driver bundled with TKGS